mmv1/third_party/terraform/acctest/bootstrap_test_utils.go

```diff
@@ -323,7 +327,7 @@ func BootstrapSharedTestNetwork(t *testing.T, testId string) string {
 		url := fmt.Sprintf("%sprojects/%s/global/networks", config.ComputeBasePath, project)
 		netObj := map[string]interface{}{
 			"name":                  networkName,
-			"autoCreateSubnetworks": false,
+			"autoCreateSubnetworks": autoCreateSubnetworks,
```

Comment:
Note that we'll never update existing networks, even if they've drifted- we may want to consider drift detection. Maybe we should additionally bootstrap the subnetwork(s), too, and avoid needing to check this at all?

Reply:
We're no longer using the input as every call is setting it to false, if that's passing tests can we remove it altogether?